### DISCUSSION TITLE
Bulk selection of Agents

### DISCUSSION THREAD

**Discussion by Kieran-staffs-tech** [amidaware/tacticalrmm#1670] [Ideas]
It would be nice to have the ability to select multiple agents for bulk actions. Things like move, delete, assign policies etc.

**Comment by zionkv** [comment]
any updates with it?

**Comment by P6g9YHK6** [comment]
not planned atm
there is an opened issue on the subject.

**Comment by iSodeik** [comment]
I wrote my own frontend connected to a backend server in between, that makes the actual calls to trmm. My Server in the middle handles multi move and multi Delete and sends them as single moves/deletes to trmm. 

A bit complicated, but is working quite nicely with drag and drop
![image](https://github.com/user-attachments/assets/88893b5e-2c30-45db-b827-322c9d7a6a69)
